packages/core/src/theme/theme.ts

```diff
@@ -521,6 +521,7 @@ export type ColorStyles = Record<ColorStyleName, ColorStyle>
 export type ColorScheme = {
   foreground: string
   background: string
+  backgroundName: string
```

Comment:
This could be typescript-y. Here's a proposal:

```typescript
type _PaletteFamily = Array<keyof PaletteFamilies>[number]
type _PaletteFamilyOption = Array<keyof PaletteFamily>[number]
export type PaletteColor = `${_PaletteFamily}.${_PaletteFamilyOption}`
```

With this, the type of `backgroundName` can be `PaletteColor`. These types might be useful in general for other props on other components too 🤔